.github/workflows/build.yml

```diff
@@ -45,7 +45,8 @@ jobs:
           version: '0.12.1'
         runs-on:
         - ubuntu-latest
-        - macos-latest
+        - macos-13 # amd64
+        - macos-14 # arm64
```

Comment:
TODO:
 - [ ] Add a target triple here and include it explicitly in the cargo install command to make sure the right target is built